### PR TITLE
Minor text fixes (fixes #289, fixes #284)

### LINKED ISSUE #284: Capitalisation inconsistent
"My Account" vs "My account"

https://www.dropbox.com/s/q9qe4qofzzu3joh/Screenshot%202015-08-06%2014.58.04.png?dl=0

### LINKED ISSUE #289: Inconsistent per month
In an email receipt its "per month", on a payment page is "per month". On the list of products in the management interface its "/ month".

https://www.dropbox.com/s/ilj3jzb2fhqr9vf/Screenshot%202015-08-06%2015.22.24.png?dl=0

Let's be consistent and use "per month" since that's easier to localize.

### PR DESCRIPTION
**Pull request by andymckay**
* fixes #289 and #284 
* and "Payment Methods" vs "Payment methods"

### CODE REVIEW

File: public/js/components/subscription.jsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ export default class Subscription extends Component {
         </div>
         <div className="price">
           <div>{productData.price.en}</div>
-          <div>/ {gettext('month')}</div>
+          <div>per {gettext('month')}</div>
```
Comment (tofumatt) [inline public/js/components/subscription.jsx]: No translation? :scream: :fr:

**Comment by muffinresearch** [comment]
r+

File: public/js/components/subscription.jsx (hunk 1 above)
Comment (muffinresearch) [inline public/js/components/subscription.jsx]: Oops I jumped the gun landing this.

Comment (muffinresearch) [inline public/js/components/subscription.jsx]: Fixed https://github.com/mozilla/payments-ui/commit/ed84f964bbb467c88012e6f57535343618c83a5c

**Pull request merged**